### PR TITLE
New blog post about components

### PR DESCRIPTION
**Pull request by comfroels**
Make sure I'm not lying about anything @kendaleiv

### CODE REVIEW

File: _posts/2017-02-23-the-component-life.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- javascript
+- frontend
+twitter_text: "The Component Life #webcomponents #javascript"
+authors: Nathan White
+image: /images/modules.jpg
+image_credit: Nathan White
+external_links_target_blank: true
+---
+
+At RimDev we're trying to rethink the way we build our frontend. We have so many properties with common functionality, and it was starting to be cumbersome (not to mention a waste of time) to implement similar features across all of our products. We knew there was a better approach. This is what brought us to a redistributable component architecure.
```
Comment (kendaleiv) [inline _posts/2017-02-23-the-component-life.md]: Can you fix the spelling on `architecure`?

File: _posts/2017-02-23-the-component-life.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+twitter_text: "The Component Life #webcomponents #javascript"
+authors: Nathan White
+image: /images/modules.jpg
+image_credit: Nathan White
+external_links_target_blank: true
+---
+
+At RimDev we're trying to rethink the way we build our frontend. We have so many properties with common functionality, and it was starting to be cumbersome (not to mention a waste of time) to implement similar features across all of our products. We knew there was a better approach. This is what brought us to a redistributable component architecure. 
+
+One of the recent components we've built is for our "Related Content." Now on the backend, it's just one simple endpoint. However, on the frontend, we have many different contexts for where/why this component is used. For example, sometimes we might utilize this component on a static blog site. Other times we might use it to show related articles inside of one of our products. We needed to build out these pieces in a very extensible manner for the frontend. There should be a minimal barrier to implementation and it should allow the frontend developer(designer) an easy workflow to embed inside another site.
```
Comment (kendaleiv) [inline _posts/2017-02-23-the-component-life.md]: Suggestion revision: "allow the frontend developer(designer)" -> "allow a frontend developer or designer".

File: _posts/2017-02-23-the-component-life.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    this.configuration = defaultConfiguration;
+
+    if (configuration) {
+      Object.assign(this.configuration, configuration);
+    }
+  }
+}
+```
+
+I would like to note the `defaultDomScope` in the configuration object. This allows you to scope the components to a given DOM element, giving even more control to the frontend implementor.
```
Comment (kendaleiv) [inline _posts/2017-02-23-the-component-life.md]: We may want to switch `implementor` to `implementer` throughout.

File: _posts/2017-02-23-the-component-life.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "The Component Life"
```
Comment (kendaleiv) [inline _posts/2017-02-23-the-component-life.md]: A more descriptive title may perform better with organic searches.

File: _posts/2017-02-23-the-component-life.md (hunk 3 above)
Comment (comfroels) [inline _posts/2017-02-23-the-component-life.md]: Haha is one british?

**Pull request merged**